### ISSUE TITLE
When ever I try to run Frontend and backend.py, they glich out every time I try to open it

### ISSUE THREAD

**Issue by unknownAlt-LA** [plutoniumx/Anigame-Sniper-Gen-2#6]

**Comment by Sebastian09-09** [comment]
right click them 
and then click on EDIT with IDLE , and then run from there ( F5 key )
it will show you the error , if you can then tell me what the error is i can help you

**Comment by unknownAlt-LA** [comment]
> right click them
> and then click on EDIT with IDLE , and then run from there ( F5 key )
> it will show you the error , if you can then tell me what the error is i can help you

![image](https://user-images.githubusercontent.com/90579050/133160151-867d0c27-3b8a-470c-af5a-3a8da7863f9a.png)

**Comment by unknownAlt-LA** [comment]
Ive been trying to import colorama, and do realize that I should already have it downloaded when I ran Lunch.bat. but somehow it still doesn't work

**Comment by unknownAlt-LA** [comment]
I just realized another problem. Somehow my laptop dose not recognize pip as a command.
![image](https://user-images.githubusercontent.com/90579050/133160716-26d55c76-3e5c-4233-b9c0-aef0561e04d0.png)

**Comment by Sebastian09-09** [comment]
yeah , so you need to install pip as well , generally when you install python it shows you a checkboxes asking if you want pip . maybe you didnt check it , now you will have to do it manually 
refer to this article : https://www.geeksforgeeks.org/how-to-install-pip-on-windows/
